### ISSUE TITLE
Why Akka.Persistence still beta?

### ISSUE THREAD

**Issue by Condor2708** [akkadotnet/akka.net#2614]

**Comment by Danthar** [comment]
Persistence is still Beta, because there is still work being done to improve the structure of the plugins. Beta status gives us the freedom to change the api where we seem fit, in order to improve performance and functionality of the overall persistence layer. For example, recently @Horusiath improved performance of all our sql based plugins.
Added to that, in order for the core team to feel comfortable to bring it out of Beta, we want to have proper CI setup for all officially supported persistence plugins. Which is something @heynickc is working at.
All this work is underway as we speak. 

Check out the activity on the https://github.com/akkadotnet/Akka.Persistence.SqlServer repository for example. 

So in short, the team is hard at work to be able to deliver something with the same level of quality, test coverage and confidence as we try to do for all Akka.net modules. 

We plan to get a stable release of the persistence modules for version 1.5

**Comment by Condor2708** [comment]
Great, thanks for the information
